pysoundcomparisons/__main__.py

```diff
@@ -99,10 +99,10 @@ def _copy_save_url(url, query, dest):
     try:
         response = urlopen(url + "/" + query)
     except:
-        print("Please check %s/%s connection" % (url, query))
+        args.log.error("Please check %s/%s connection" % (url, query))
```

Comment:
`args` is **not** known inside this function! It could be passed in as additional argument, but I think it's better practice to limit log calls to the first layer of functions, i.e. to the functions implementing commands. Otherwise, user-facing messages are easily spread all over the code base.